api/v1alpha1/azurepublicipaddress_types.go

```diff
@@ -23,6 +24,8 @@ type AzurePublicIPAddressSpec struct {
 	IdleTimeoutInMinutes     int    `json:"idleTimeoutInMinutes"`
 	PublicIPAddressVersion   string `json:"publicIPAddressVersion"`
 	SkuName                  string `json:"skuName"`
+	// +optional
+	IPTags map[string]string `json:"ipTags,omitempty"`
```

Comment:
@frodopwns - should we regenerate the helm pkg stuff now or do we just do it before we ship?